azavea/temperate
# Style header and footer

## Overview

Style header, including user dropdown menu. Make responsive.

Style footer. Make responsive.

Added grant info to footer, per discussion with @jcahail.

### Demo

#### Header desktop (app)
![image](https://user-images.githubusercontent.com/128699/36734476-98a95238-1ba1-11e8-94cf-08e0da7c96b8.png)

#### User dropdown
![image](https://user-images.githubusercontent.com/128699/36734522-b869984e-1ba1-11e8-971f-34db2c641837.png)

#### Header desktop (marketing)
![screen shot 2018-02-27 at 9 49 55 am](https://user-images.githubusercontent.com/128699/36735215-a0734580-1ba3-11e8-8186-81bcced0d5fd.png)

#### Header mobile (marketing)
![screen shot 2018-02-27 at 9 42 08 am](https://user-images.githubusercontent.com/128699/36734945-f759d05e-1ba2-11e8-950e-ef8f965a5621.png)

#### Footer desktop (app)
![screen shot 2018-02-27 at 9 46 04 am](https://user-images.githubusercontent.com/128699/36734981-14c1488e-1ba3-11e8-8361-1177be81ac8a.png)

#### Footer mobile (marketing: no "About")
![screen shot 2018-02-27 at 9 45 15 am](https://user-images.githubusercontent.com/128699/36734951-fb1c1d64-1ba2-11e8-85df-27fdf57c9a37.png)


### Notes

The `routerLink` attributes of the nav buttons in both header and footer may need some attention.

The "About" link in footer should only appear while in the app, not on the marketing page. Clicking it should land on the marketing page.


## Testing Instructions

 * `pull` and `server`
 * Go to marketing page, logged out. Confirm header and footer look per above.
 * Resize to narrow viewport. Confirm header and footer look per above.
 * Login. Confirm header and footer look per above.
 * Open user dropdown, hover on its items. Confirm it looks per above.

Closes #546. 
Closes #547.
